config/__init__.py

```diff
@@ -278,7 +278,7 @@
 TENTACLE_PACKAGE_DESCRIPTION = "package_description"
 EVALUATOR_CONFIG_FOLDER = "config"
 TENTACLES_TRADING_MODE_PATH = "Mode"
-TENTACLES_DEFAULT_BRANCH = "dev"
+TENTACLES_DEFAULT_BRANCH = SHORT_VERSION
```

Comment:
"master"

Reply:
No, I create a branch 0.3.3 on tentacles that is up to date from master. 
If we use master here, it will be impossible to install this version after some releases.